### PR TITLE
feat(components/indicators): use more descriptive default ARIA labels for tokens and announce when they are dismissed to screen readers

### PR DESCRIPTION
**Pull request by Blackbaud-CoreyArcher**
[AB#2670094](https://dev.azure.com/blackbaud/f565481a-7bc9-4083-95d5-4f953da6d499/_workitems/edit/2670094)

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/1865/storybook/)

Component Storybooks:

- [action-bars-storybook](https://blackbaud.github.io/skyux-pr-preview/1865/storybooks/action-bars-storybook/)
- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/1865/storybooks/ag-grid-storybook/)
- [angular-tree-component-storybook](https://blackbaud.github.io/skyux-pr-preview/1865/storybooks/angular-tree-component-storybook/)
- [avatar-storybook](https://blackbaud.github.io/skyux-pr-preview/1865/storybooks/avatar-storybook/)
- [colorpicker-storybook](https://blackbaud.github.io/skyux-pr-preview/1865/storybooks/colorpicker-storybook/)
- [data-manager-storybook](https://blackbaud.github.io/skyux-pr-preview/1865/storybooks/data-manager-storybook/)
- [datetime-storybook](https://blackbaud.github.io/skyux-pr-preview/1865/storybooks/datetime-storybook/)
- [errors-storybook](https://blackbaud.github.io/skyux-pr-preview/1865/storybooks/errors-storybook/)
- [flyout-storybook](https://blackbaud.github.io/skyux-pr-preview/1865/storybooks/flyout-storybook/)
- [forms-storybook](https://blackbaud.github.io/skyux-pr-preview/1865/storybooks/forms-storybook/)
- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/1865/storybooks/indicators-storybook/)
- [inline-form-storybook](https://blackbaud.github.io/skyux-pr-preview/1865/storybooks/inline-form-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/1865/storybooks/layout-storybook/)
- [lists-storybook](https://blackbaud.github.io/skyux-pr-preview/1865/storybooks/lists-storybook/)
- [lookup-storybook](https://blackbaud.github.io/skyux-pr-preview/1865/storybooks/lookup-storybook/)
- [modals-storybook](https://blackbaud.github.io/skyux-pr-preview/1865/storybooks/modals-storybook/)
- [navbar-storybook](https://blackbaud.github.io/skyux-pr-preview/1865/storybooks/navbar-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/1865/storybooks/pages-storybook/)
- [phone-field-storybook](https://blackbaud.github.io/skyux-pr-preview/1865/storybooks/phone-field-storybook/)
- [popovers-storybook](https://blackbaud.github.io/skyux-pr-preview/1865/storybooks/popovers-storybook/)
- [progress-indicator-storybook](https://blackbaud.github.io/skyux-pr-preview/1865/storybooks/progress-indicator-storybook/)
- [split-view-storybook](https://blackbaud.github.io/skyux-pr-preview/1865/storybooks/split-view-storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/1865/storybooks/tabs-storybook/)
- [text-editor-storybook](https://blackbaud.github.io/skyux-pr-preview/1865/storybooks/text-editor-storybook/)
- [theme-storybook](https://blackbaud.github.io/skyux-pr-preview/1865/storybooks/theme-storybook/)
- [tiles-storybook](https://blackbaud.github.io/skyux-pr-preview/1865/storybooks/tiles-storybook/)
- [toast-storybook](https://blackbaud.github.io/skyux-pr-preview/1865/storybooks/toast-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/1865/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/1865/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/1865/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/1865/playground/)

File: libs/components/indicators/src/lib/modules/tokens/token.component.spec.ts [hunk 1]
```diff
@@ -95,7 +101,10 @@ describe('Token component', () => {
       component.ariaLabel = undefined;
       fixture.detectChanges();
 
-      expect(btnEl.getAttribute('aria-label')).toBe('Remove item');
+      expect(btnEl.getAttribute('aria-label')).toBeNull();
```
Comment (Blackbaud-TrevorBurch) [inline libs/components/indicators/src/lib/modules/tokens/token.component.spec.ts:105]: Minor - but could we add a check along with the expectation above to make sure `aria-labelledby` is null when `aria-label` is `test`?

**Review by Blackbaud-TrevorBurch — COMMENTED** [review]
One small thing

File: libs/components/indicators/src/lib/modules/tokens/token.component.spec.ts (hunk 1 above)
Comment (Blackbaud-CoreyArcher) [reply libs/components/indicators/src/lib/modules/tokens/token.component.spec.ts:105]: added

**Review by Blackbaud-TrevorBurch — DISMISSED** [review]

**Review by Blackbaud-TrevorBurch — DISMISSED** [review]

**Review by Blackbaud-TrevorBurch — APPROVED** [review]

**Pull request merged**